### DISCUSSION TITLE
斯坦福 CS 149 并行计算 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#44] [Announcements]
# 斯坦福 CS 149 并行计算 | HackWay技术学习路线

课程名称：  Parallel Computing

https://hackway.org/docs/cs/elective/parallel/cs149

**Comment by shenzheyu** [comment]
2022年春的链接错了

**Reply by lidongyx** [reply]
链接现在已经更新了，并整理了关于这门课的一些信息，谢谢支持

**Reply by shenzheyu** [reply]
感谢你的整理
